x-pack/plugins/fleet/public/applications/fleet/components/fleet_server_instructions/hooks/use_latest_fleet_servers.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import { useCallback, useRef, useState, useEffect } from 'react';
+
```

Comment:
Did you mean to change this fleet file?

Reply:
hmm that's weird.. I don't remember opening this but yeah I'll revert this change. Thanks!

Reply:
Seems to have come from an auto-commit 0d8774d7a5bc6313a0c80881ac6cd1ddd622e3e1

Reply:
Since it came from the auto-commit I won't revert the change.